### ISSUE TITLE
Data.js only renders in Dev environment

### ISSUE THREAD

**Issue by JDRomano2** [JDRomano2/venomkb#2]
Since refactoring the DevTools.js container, the pages for individual entities won't render. As of dc07a800c4f656553b5be7cb8fe67b1cf8a46b34, this issue has existed for at least a few commits.

My best guess is that this is the result of how we choose to render DevTools. The first step to try resolving this is to follow the instructions at https://github.com/reduxjs/redux-devtools/blob/master/docs/Walkthrough.md#exclude-devtools-from-production-builds. If that doesn't work, I'll work backwards.

**Comment by JDRomano2** [comment]
Errors in the browser look like the following:

```
main.min.js:36 Error: Actions must be plain objects. Use custom middleware for async actions.
    at m (main.min.js:19)
    at DataDetailContainer.value (main.min.js:50)
    at commitLifeCycles (main.min.js:36)
    at C (main.min.js:36)
    at E (main.min.js:36)
    at _ (main.min.js:36)
    at g (main.min.js:36)
    at m (main.min.js:36)
    at c (main.min.js:36)
    at t (main.min.js:36)
br @ main.min.js:36
commitErrorLogging @ main.min.js:36
C @ main.min.js:36
E @ main.min.js:36
_ @ main.min.js:36
g @ main.min.js:36
m @ main.min.js:36
c @ main.min.js:36
t @ main.min.js:36
updateContainer @ main.min.js:36
_o.render @ main.min.js:36
(anonymous) @ main.min.js:36
unbatchedUpdates @ main.min.js:36
xo @ main.min.js:36
...
main.min.js:20 
```

```
Unhandled promise rejection Error: Actions must be plain objects. Use custom middleware for async actions.
    at m (main.min.js:19)
    at DataDetailContainer.value (main.min.js:50)
    at commitLifeCycles (main.min.js:36)
    at C (main.min.js:36)
    at E (main.min.js:36)
    at _ (main.min.js:36)
    at g (main.min.js:36)
    at m (main.min.js:36)
    at c (main.min.js:36)
    at t (main.min.js:36)
```

**Comment by JDRomano2** [comment]
Seemingly fixed by 01eb285480d82d3dcd4dda54f0b0e3747c6f6742 (my intuition was correct).
